### PR TITLE
User Dashboard

### PR DESCRIPTION
**Pull request by MalcolmAG**
Easily extendible to entrepreneur profile. 
Mockup. Images are currently stored in src/staticImages
Dashboard currently has 4 components. A Navbar, UserInfo sidebar, ProjectCard for each project, and the parent UserDashboard

### CODE REVIEW

File: client/src/pages/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  appBar: {
+    background: "white",
+    color: "black",
+    elevation: 0,
+    "&MuiPaper": {
+      elevation: 0,
+    },
+  },
+  logo: {
+    width: theme.spacing(4),
```
Comment (bonnieli) [inline client/src/pages/Dashboard.js:70]: nice great use of spacings!

File: client/src/pages/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    color: "black",
+  },
+  appBar: {
+    background: "white",
+    color: "black",
+    elevation: 0,
+    "&MuiPaper": {
```
Comment (bonnieli) [inline client/src/pages/Dashboard.js]: hm this looks interesting, is this working? thought you might need a space inbetween like `& .MuiPaper` or something

File: client/src/pages/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+        </Container>
+      </Box>
+    </Box>
+  );
+}
+
+function ProjectCard(props) {
+  const classes = useStyles();
+  console.log(props.ver);
+  const projectStatic = [
```
Comment (bonnieli) [inline client/src/pages/Dashboard.js]: you might want to move this static projects list outside of this container, and map through each object and pass it on as a prop

**Review by bonnieli — COMMENTED** [review]
overall looks good

File: client/src/pages/Dashboard.js (hunk 3 above)
Comment (MalcolmAG) [reply client/src/pages/Dashboard.js]: That's a good idea. I'll make that change

File: client/src/pages/Dashboard.js (hunk 1 above)
Comment (MalcolmAG) [reply client/src/pages/Dashboard.js:70]: Thank you

File: client/src/pages/Dashboard.js (hunk 2 above)
Comment (MalcolmAG) [reply client/src/pages/Dashboard.js]: That is actually unnecessary, forgot to remove it

**Review by bonnieli — APPROVED** [review]

**Review by tmclean15 — APPROVED** [review]
The dashboard looks great so far!

**Pull request merged**